### PR TITLE
Improve plugin card hyperlinks

### PR DESCRIPTION
**Pull request by jsiegenthaler**
## :recycle: Current situation
The current Info, Verified and Funding icons are very small as clickable icons on a small iPhone display. This makes them difficult to hit, especially for users with larger fingers.

## :bulb: Proposed solution
The changes expand the hyperlink anchor to the icon and text, so the user can tap the text or icon to access the feature.
The name of the plugin goes to the same hyperlink destination as the Info icon. Colour could be better though

## :heavy_plus_sign: Additional Information
Can someone improve the colour of the now clickable plugin name?

### CODE REVIEW

File: ui/src/app/modules/plugins/plugin-card/plugin-card.component.html [hunk 1]
```diff
@@ -5,36 +5,40 @@
...
               <i class="fas fa-fw fa-shield-alt fa-lg" style="opacity: 0.25;"></i>
-            </a> {{ 'plugins.donate.button_not_verified' | translate }}
+              {{ 'plugins.donate.button_not_verified' | translate }}
+            </a> 
           </p>
           <p class="card-text mb-2">
             <a href="javascript:void(0)" class="card-link pink-text" *ngIf="plugin.verifiedPlugin && plugin.funding"
                (click)="openFundingModal(plugin)">
               <i class="fas fa-fw fa-heart fa-lg"></i>
+              <span *ngIf="!plugin.verifiedPlugin || !plugin.funding">
```
Comment (bwp91) [inline ui/src/app/modules/plugins/plugin-card/plugin-card.component.html]: This won't work here.
The condition on the `<a` is that `*ngIf="plugin.verifiedPlugin && plugin.funding"` so having the `<span *ngIf="!plugin.verifiedPlugin || !plugin.funding">` inside will not work

In this case the span needs to be kept outside the <a>

Comment (jsiegenthaler) [reply ui/src/app/modules/plugins/plugin-card/plugin-card.component.html]: Thanks, I've fixed it and will update shortly

Comment (jsiegenthaler) [reply ui/src/app/modules/plugins/plugin-card/plugin-card.component.html]: Fixed with the commit from today, Dec 9, 2023.

**Comment by mkz212** [comment]
@jsiegenthaler I didn't notice this PR and added PR myself with many fixes to the plugin card, including this one.

**Pull request merged**